grpc/gcp/s2a/common.proto

```diff
@@ -0,0 +1,79 @@
+// Copyright 2022 Google LLC
```

Comment:
Is this right? This project typically has gRPC Authors with the copyright. With Google as the copyright holder it is sort of like "why is this in the grpc package?"

Reply:
Updated the files to use gRPC Authors with the copyright.